docs/runtime_suite/mailchimp-notification-service/10_overview.md

```diff
@@ -0,0 +1,175 @@
+---
+id: overview
+title: Mailchimp/Mandrill Notification Service
+sidebar_label: Overview
+---
+
+<!--
+WARNING: this file was automatically generated by Mia-Platform Doc Aggregator.
+DO NOT MODIFY IT BY HAND.
+Instead, modify the source file and run the aggregator to regenerate this file.
+-->
+
+The **Mailchimp/Mandrill Notification Service** is a microservice responsible to interact with Mailchimp and Mandrill for:
+
+* **Sending transactional emails**: using Mailchimp defined email templates and Mandrill transactional APIs
+* **Interact with Mailchimp audiences**: subscription, deletion, user status update
+
+# 1) Sending transactional emails
+Transactional emails service is provided with 2 main endpoints:
+
+* **Synchronous**: waits for Mailchimp/Mandrill to have actually queued the transactional email request
+
+* **Asynchronous**: fire and forget. The endpoint immediately returns, the request is sent to Mailchimp/Mandrill but the service will not wait for Mailchimp/Mandrill response
+
+## Synchronous transactional email send
+The service is provided through the endpoint `POST /transactional/send-template`.
+
+An example JSON body is presented below:
+```
+{
+	"templateName": "my_template",
+	"subject": "Hello world!",
+	"fromEmail": "example@test.com",
+	"fromName": "Visible subject name",
+	"templateMergeVars": [
+		{
+            "name": "HELLO_WORLD",
+            "content": "I can provide the content I want here"
+		},
+	 ],
+     "to": [
+           {
+               "email": "receiver@test.com",
+               "type": "to"
+           },
+           {
+               "email": "cc@test.com",
+               "type": "cc"
+           },
+           {
+               "email": "bcc@test.com",
+               "type": "bcc"
+           }
+     ],
+     "attachments": [
+        {
+           "mimeType": "application/pdf",
+           "visibleName": "example.pdf",
+           "base64Content": "base64encodedContent",
+        }
+     ]
+}
+```
+
+* `templateName`: name of the email template as created in Mailchimp `Content -> Email Templates` section. 
+    * NB: a `Send To Mandrill` action on the template from the Mailchimp UI is required to be able to send transactional emails using the template defined
+* `subject`: Email subject
+* `fromEmail`: this is the email sender address
+* `fromName`: this is the email friendly sender name
+* `templateMergeVars`: array body parameter
+    * each array entry defines an email `MERGE TAG`: in case the transactional email requires custom parameters inside its body, those can be provided using the `templateMergeVars` array
```

Comment:
```suggestion
* `templateName`: name of the email template as created in Mailchimp `Content -> Email Templates` section. 
    * NB: a `Send To Mandrill` action on the template from the Mailchimp UI is required to be able to send transactional emails using the template defined
* `subject`: Email subject
* `fromEmail`: this is the email sender's address
* `fromName`: this is the email friendly sender name
* `templateMergeVars`: array body parameter
    * each array entry defines an email `MERGE TAG`: in case the transactional email requires custom parameters inside its body, those can be provided using the `templateMergeVars` array
```